Doc: update the documentation of addKeyword args (#2090)
epoberezkin/ajv: docs/api.md

<a name="api-addkeyword"></a>

> ### ajv.addKeyword(definition: string | KeywordDefinition): Ajv

Reviewer: ```suggestion
### ajv.addKeyword(definition: string | object): Ajv
```

}
```

> If only the property `keyword` is provided in the definition object, one can also pass the string value alone to the method as the argument.

Reviewer: ```suggestion
If only the property `keyword` is provided in the definition object, you can also pass the keyword name as the argument.
```
